core/src/main/java/org/apache/carbondata/core/indexstore/BlockletDataMapIndexStore.java

```diff
@@ -244,4 +247,22 @@ public void clearAccessCount(
       cacheable.clear();
     }
   }
+
+  public static class BlockMetaInfo {
```

Comment:
Is it only used in this class? I think it is better to move to outside if not just used in this class

Reply:
ok